Documentation/PageTsconfig/Mod/WebList.rst

```diff
@@ -528,15 +563,16 @@ noExportRecordsLinks
     :type:  boolean
     :Default: 0
 
-    If set, the :guilabel:`Download` and :guilabel:`Export` buttons are hidden
-    in the list module. This applies to
-    the :guilabel:`Export` button located at the top left for t3d exports, the
-    :guilabel:`Download` button directly on the table
-    listing for csv download and the :guilabel:`Download` button in the tables
-    single view.
+    ..  versionchanged:: 13.1.0, 12.4.11
+        Until the mentioned versions this options also hid the the
+        :guilabel:`Download` button used for CSV export. Use
```

Comment:
Solved "hid" and duplicate "the", but I don't know what you want to statr writing after "Use..."

```suggestion
        Until the mentioned versions this options also hide the
        :guilabel:`Download` button used for CSV export. Use
```

Reply:
hid is past tense of hide, I agree about removing the duplicate the